packages/amplify-graphql-schema-generator/vpc-db-lambda/src/aws-rds-global-bundle.pem

```diff
@@ -0,0 +1,3052 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
I don't recommend storing these certificates in the library code, because of the following reasons:
- If these certificates were to be refreshed, it would require **all** our customers to upgrade their dependency to the `latest` which can be painful.
- There is no edit protection for these files in terms of external contributions, so if we accidentally merge a PR that updates these, that breaks our customers.

The second can be mitigated using `CODEOWNERS` in Github, but want to know if we thought through the first.

Reply:
For your first point, this file contains only the certificates provided by RDS service. They don't change often, the default provided certificate `rds-ca-rsa2048-g1` is valid for 40 years and (`rds-ca-rsa4096-g1` and `rds-ca-ecc384-g1`) are valid for 100 years.

If RDS is updating their global CA, we should be okay to push it on a next release. Since these don't change often, I don't think it is a problem we need to worry about right now.

We can consider pulling these certificates dynamically at runtime but I don't want to do this now unless we clearly understand all the side effects of that approach.

For your second point, I agree, we can add it to the `codeowners` file.

Reply:
> We can consider pulling these certificates dynamically at runtime but I don't want to do this now unless we clearly understand all the side effects of that approach.

Lets discuss this option and agree we can do this without breaking changes in subsequent releases.

> For your second point, I agree, we can add it to the codeowners file.

Do you want to make this change in the PR?